### PR TITLE
Gen3 Monthly Release 2022.03 qa-heal.planx-pla.net 1646242662

### PR DESCRIPTION
**Pull request by PlanXCyborg**
Applying version 2022.03 to qa-heal.planx-pla.net

### CODE REVIEW

File: qa-heal.planx-pla.net/manifest.json [hunk 1]
```diff
@@ -8,27 +8,27 @@
...
+    "pidgin": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/pidgin:2022.03",
+    "portal": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/data-portal:2022.03",
+    "revproxy": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/nginx:2022.03",
+    "sheepdog": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/sheepdog:2022.03",
+    "ssjdispatcher": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/ssjdispatcher:2022.03",
+    "sower": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/sower:2022.03",
+    "spark": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/gen3-spark:2022.03",
+    "tube": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/tube:2022.03",
+    "wts": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/workspace-token-service:2022.03",
+    "frontend-framework": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/frontend-framework:2022.03"
```
Comment (haraprasadj) [inline qa-heal.planx-pla.net/manifest.json:31]:
```suggestion
    "frontend-framework": "quay.io/cdis/frontend-framework:main"
```

File: qa-heal.planx-pla.net/manifest.json [hunk 2]
```diff
@@ -8,27 +8,27 @@
...
-    "ssjdispatcher": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/ssjdispatcher:2022.02",
-    "sower": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/sower:2022.02",
-    "spark": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/gen3-spark:2022.02",
-    "tube": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/tube:2022.02",
-    "wts": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/workspace-token-service:2022.02",
-    "frontend-framework": "quay.io/cdis/frontend-framework:main"
+    "hatchery": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/hatchery:2022.03",
+    "indexd": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/indexd:2022.03",
+    "manifestservice": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/manifestservice:2022.03",
+    "metadata": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/metadata-service:2022.03",
```
Comment (haraprasadj) [inline qa-heal.planx-pla.net/manifest.json:20]:
```suggestion
    "metadata": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/metadata-service:1.6.4",
```

File: qa-heal.planx-pla.net/manifest.json [hunk 3]
```diff
@@ -225,7 +225,7 @@
       "activeDeadlineSeconds": 600,
       "container": {
         "name": "job-task",
-        "image": "quay.io/cdis/batch-export:3.0.0",
+        "image": "quay.io/cdis/batch-export:2022.03",
```
Comment (haraprasadj) [inline qa-heal.planx-pla.net/manifest.json:228]:
```suggestion
        "image": "quay.io/cdis/batch-export:3.0.0",
```

**Comment by PlanXCyborg** [comment]
# qa-heal.planx-pla.net/manifest.json
## :warning: Services on branch
- frontend-framework

File: qa-heal.planx-pla.net/manifest.json [hunk 4]
```diff
@@ -8,26 +8,26 @@
...
-    "revproxy": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/nginx:2022.02",
-    "sheepdog": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/sheepdog:2022.02",
-    "ssjdispatcher": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/ssjdispatcher:2022.02",
-    "sower": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/sower:2022.02",
-    "spark": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/gen3-spark:2022.02",
-    "tube": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/tube:2022.02",
-    "wts": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/workspace-token-service:2022.02",
+    "peregrine": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/peregrine:2022.03",
+    "pidgin": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/pidgin:2022.03",
+    "portal": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/data-portal:2022.03",
```
Comment (mfshao) [inline qa-heal.planx-pla.net/manifest.json:23]: can you check with @mcannalte to see if he still needs this

Comment (mcannalte) [reply qa-heal.planx-pla.net/manifest.json:23]: Thanks for checking. I don't need this at the moment

**Review by mfshao — APPROVED** [review]

File: qa-heal.planx-pla.net/manifest.json (hunk 4 above)
Comment (haraprasadj) [reply qa-heal.planx-pla.net/manifest.json:23]: I will notify on slack once the PR passes that I would be deploying 2022.03 images for testing. We can switch again after doing a sanity check.

**Pull request merged**